src/System-Sources/RemoteString.class.st

```diff
@@ -1,18 +1,19 @@
 "
 My instances provide an external file reference to a piece of text.  It may be the sourceCode of a method, or the class comments of a class.
 
-The changes file or file-in file usually has a chunk that is just the source string of a method:
+The 'changes' file or 'file-in' file usually has a chunk that is just the source string of a method:
```

Comment:
I'm not sure whether the changes and file-in files are meant to be different here. If they are, i would say "changes and file-in files usually have..."; if they aren't, i would say "the changes (or file-in) file usually has..."